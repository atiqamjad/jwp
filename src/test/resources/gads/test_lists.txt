Begin unordered list #1
<li>item1 of the unordered list #1.
<li>item2

of the unordered list #1?</li>
<li>item3 of the unordered list #1!</li><li>  item4 of the unordered list #1

<li>item5 of the unordered list #1;
<li>item6 of the unordered list #1$<li>item7 of the unordered list #1<p>paragraph one two three four five</p>

Begin unordered list #2<ul>
<li>one</li><li>two</li>
</ul>

Begin unordered list #3
  word<li>one</li><li>two</li>
</ul>
<p>
Begin ordered list #1</p>
1)item1 of the ordered list #1
2)   item2 of the ordered list #1
3)item3 of    the ordered list #1
4)item4 of the ordered list #1

Begin ordered list #2
1.item1 of the ordered list #2
2.item2 of the ordered list #2
3.item3 of the ordered list #2
4.item4 of the ordered list #2


OEF