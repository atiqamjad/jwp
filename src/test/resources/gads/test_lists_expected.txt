Begin unordered list #1
<ul>
<li>item1 of the unordered list #1</li>
<li>item2 of the unordered list #1</li>
<li>item3 of the unordered list #1</li>
<li>item4 of the unordered list #1</li>
<li>item5 of the unordered list #1</li>
<li>item6 of the unordered list #1</li>
<li>item7 of the unordered list #1</li>
</ul>

Begin ordered list #1
<ol>
<li>item1 of the ordered list #1</li>
<li>item2 of the ordered list #1</li>
<li>item3 of the ordered list #1</li>
<li>item4 of the ordered list #1</li>
</ol>

Begin ordered list #2
<ol>
<li>item1 of the ordered list #2</li>
<li>item2 of the ordered list #2</li>
<li>item3 of the ordered list #2</li>
<li>item4 of the ordered list #2</li>
</ol>